pages/guides/how-to-create-tradingview-charts.mdx

```diff
@@ -111,9 +111,11 @@ The TradingView integration is a web-based widget that allows users to view Pyth
 
 6. There is a fully working open-source example of the TradingView integration by one of Pyth's contributors [here](https://github.com/cctdaniel/pyth-tv-example). The example application is deployed [here](https://pyth-tv-example.vercel.app/).
 
+Note: The TradingView plug-and-play widget does not allow for much customization. If you want to customize the widget, you can use the [TradingView Charting Library](https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/). Please see the next section for more details.
+
 ## Using Datafeed URL with Charting Library
 
-We also provide a datafeed URL that follows the [TradingView Data Integration spec](https://www.tradingview.com/rest-api-spec/#tag/Data-Integration). You can implement your own datafeed utilizing the API or use the built-in [UDF](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF) adapter. If you need a step-by-step guide, refer to the [How to connect data via Datafeed API](https://www.tradingview.com/charting-library-docs/latest/tutorials/implement_datafeed_tutorial/) tutorial.
+We also provide a UDF-compatible URL that follows the [TradingView UDF spec](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF). You can implement your own datafeed utilizing the API or use the built-in [UDF](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF) adapter with the API. If you need a step-by-step guide, refer to the [How to connect data via Datafeed API](https://www.tradingview.com/charting-library-docs/latest/tutorials/implement_datafeed_tutorial/) tutorial, or you can reference the example [here](https://github.com/cctdaniel/pyth-tv-charting-lib)
```

Comment:
same url is linked twice for UDF. suggest dropping the second one.

Reply:
fwiw the tradingview tutorial above is long and hard to follow. It doesn't give the reader the impression that they can copy/paste some code and get going fast. This doesn't need to be fixed in this PR, but we may want to make our own tutorial that's specific to the Pyth integration.

(i think you have the code here https://github.com/cctdaniel/pyth-tv-charting-lib/blob/main/src/utils/datafeed.js , and then people just need to copy that, plus the tradingview chart around it)

Reply:
by "tutorial" do you mean literally a step-by-step example? the repo that I've linked to is the "tutorial" that I was thinking of where users can just copy the files and get it working, maybe I should expand more on that to make it clearer to the users?